app/web/src/features/donations/Donations.tsx

```diff
@@ -158,11 +149,16 @@ export default function Donations() {
           )}
           variant="body2"
         >
-          {BENEFACTOR_CONTACT1}{" "}
-          <Link className={classes.link} href={"mailto:" + BENEFACTOR_EMAIL}>
-            {BENEFACTOR_EMAIL}
-          </Link>{" "}
-          {BENEFACTOR_CONTACT2}
+          <Trans t={t} i18nKey="benefactor_contact">
+            If you wish to contribute over $1000, please contact us at
+            <Link
+              className={classes.link}
+              href={`mailto:${t("benefactor_email")}`}
+            >
+              {{ email: t("benefactor_email") }}
+            </Link>
+            for us to arrange a lower fee transfer.
+          </Trans>
```

Comment:
This may look like we're hardcoding the message again and it's a bit confusing, but it's not and only used for figuring out what to swap in from this template string in the language JSON file: `"If you wish to contribute over $1000, please contact us at <1>{{email}}</1> for us to arrange a lower fee transfer."`

It's needed since we are mixing React/HTML components with the sentence, so the `Trans` component takes this "hardcoded template" and break into 3 children: the hardcoded text on the left, the `Link` and the hardcoded text on the right. It then figures out the actual text to swap/substitute in based on what's to the left/right of `<1>{{email}}</1>` (1 since `Link` is the second child element to the `Trans` component).

From looking at the examples, it seems like the best practice is to leave the English version of the text as template here and change the value in the JSON accordingly, so it gives us a rough idea of what to expect on screen.